dockers/docker-fpm-frr/bgpd.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+addr="127.0.0.1"
+port=2601
+
+while getopts ":A:P:" opt; do
+    case "${opt}" in
+        A) addr=${OPTARG}
+            ;;
+        P) port=${OPTARG}
+            ;;
+    esac
+done
+
+timeout 5s bash -c -- "until </dev/tcp/${addr}/${port}; do sleep 0.1;done"
```

Comment:
what if timeout? I think we should print error msg?

Reply:
Added a check for exit code and log a message if failed.